In [1]:
items= {1001:{"Name":"Pasta", "Price":70, "Quantity":30},
        1002:{"Name":"Rice", "Price":50, "Quantity":41},
        1003:{"Name":"Bread", "Price":60, "Quantity":10},
        1004:{"Name":"Porridge oats", "Price":40, "Quantity":15},
        1005:{"Name":"Unsweetened granola", "Price":70, "Quantity":16},
        1006:{"Name":"Salt", "Price":52, "Quantity":74},
        1007:{"Name":"Basil", "Price":45, "Quantity":12},
        1008:{"Name":"Oregano", "Price":55, "Quantity":14},
        1009:{"Name":"Cumin", "Price":85, "Quantity":40},
        1010:{"Name":"Honey", "Price":47, "Quantity":52},
        1011:{"Name":"Vinegar", "Price":14, "Quantity":41},
        1012:{"Name":"Sugar", "Price":74, "Quantity":52},
        1013:{"Name":"Cooking oil", "Price":41, "Quantity":41},
        1014:{"Name":"Butter", "Price":74, "Quantity":12},
        1015:{"Name":"Milk", "Price":42, "Quantity":32},
        1016:{"Name":"Eggs", "Price":52, "Quantity":25},
        1017:{"Name":"Cheese", "Price":75, "Quantity":32},
        1018:{"Name":"Yogurt", "Price":92, "Quantity":41},
        1019:{"Name":"Pulses", "Price":77, "Quantity":60},
        1020:{"Name":"Red kidney beans", "Price":71, "Quantity":32},
        1021:{"Name":"White beans", "Price":14, "Quantity":52},
        1022:{"Name":"Green lentils", "Price":28, "Quantity":63},
        1023:{"Name":"Chopped tomatoes", "Price":75, "Quantity":14},
        1024:{"Name":"nuts", "Price":71, "Quantity":62},
        1025:{"Name":"Noodles", "Price":75, "Quantity":31},
        1026:{"Name":"Frozen Corn", "Price":62, "Quantity":23},
        1027:{"Name":"Frozen greenpeas", "Price":98, "Quantity":62},
        1028:{"Name":"Olive Oli", "Price":42, "Quantity":35},
        1029:{"Name":"Chocolate Syrup", "Price":75, "Quantity":61},
        1030:{"Name":"Jam", "Price":42, "Quantity":34},

}

##### Adding New items to Inventory

In [2]:
product_id=int(input("Enter Product Id:"))
name=str(input("Enter Product Name:"))
price=int(input("Enter Product Price:"))
quant=int(input("Enter Product Quantity:"))

Enter Product Id:1031
Enter Product Name:Coke
Enter Product Price:40
Enter Product Quantity:50


##### Updating the inventory with newly added products

In [3]:
items[product_id]={"Name":name, "Price":price, "Quantity":quant}

In [4]:
items

{1001: {'Name': 'Pasta', 'Price': 70, 'Quantity': 30},
 1002: {'Name': 'Rice', 'Price': 50, 'Quantity': 41},
 1003: {'Name': 'Bread', 'Price': 60, 'Quantity': 10},
 1004: {'Name': 'Porridge oats', 'Price': 40, 'Quantity': 15},
 1005: {'Name': 'Unsweetened granola', 'Price': 70, 'Quantity': 16},
 1006: {'Name': 'Salt', 'Price': 52, 'Quantity': 74},
 1007: {'Name': 'Basil', 'Price': 45, 'Quantity': 12},
 1008: {'Name': 'Oregano', 'Price': 55, 'Quantity': 14},
 1009: {'Name': 'Cumin', 'Price': 85, 'Quantity': 40},
 1010: {'Name': 'Honey', 'Price': 47, 'Quantity': 52},
 1011: {'Name': 'Vinegar', 'Price': 14, 'Quantity': 41},
 1012: {'Name': 'Sugar', 'Price': 74, 'Quantity': 52},
 1013: {'Name': 'Cooking oil', 'Price': 41, 'Quantity': 41},
 1014: {'Name': 'Butter', 'Price': 74, 'Quantity': 12},
 1015: {'Name': 'Milk', 'Price': 42, 'Quantity': 32},
 1016: {'Name': 'Eggs', 'Price': 52, 'Quantity': 25},
 1017: {'Name': 'Cheese', 'Price': 75, 'Quantity': 32},
 1018: {'Name': 'Yogurt', 'Price': 

##### Item 1031 Had been Added to the Inventory

##### Updating the records.json file with newly added products

In [5]:
import json
items= json.dumps(items)
fd=open("records.json","w")
fd.write(items)
fd.close()